website/static/index.html

```diff
@@ -29,6 +29,11 @@
         .cc-revoke {
             display: none !important;
         }
+        @media (prefers-color-scheme: dark) {
```

Comment:
This should prevent the flash that dark mode users experience when navigating to other pages. I believe the JS was loading too slowly on Firefox, which made the flash noticeable.